src/js/common/containers/MHVApp.jsx

```diff
@@ -91,33 +93,28 @@ export class MHVApp extends React.Component {
       return <AcceptTermsPrompt terms={terms} cancelPath="/health-care/" onAccept={this.props.acceptTerms}/>;
     }
 
-    if (!this.isAccessible()) {
+    if (!this.hasService()) {
```

Comment:
Made a distinction here where `hasAccount` determines whether we should attempt account creation, and `hasService` determines whether the user has access.